weblab/entities/processing.py

```diff
@@ -160,7 +160,7 @@ def process_check_protocol_callback(data):
 def record_experiments_to_run(user, entity, commit):
     """Record what experiments to run automatically on a new entity version.
 
-    Any experiments that were run with the parent version(s) (that are visible to the user)
+    The latest experiments that were run with the parent version(s) (that are visible to the user)
```

Comment:
I think it's worth expanding this description slightly to be more detailed, based on what's in https://github.com/ModellingWebLab/WebLab/issues/230#issuecomment-705484207. Would you agree?